dev/Deployment/DeploymentManager.cpp

```diff
@@ -357,36 +391,50 @@ namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implem
         return std::wstring{ path.get() };
     }
 
-    // Adds the package at the path using PackageManager.
+    // If registerHigherVersionPackage == false, Adds the current version package at the passed in path using PackageManager.
+    // If registerHigherVersionPackage == true, Registers the higher version package using the passed in path as manifest path and PackageManager.
```

Comment:
I think we discussed about it in person and concluded that we should add package instead of registering existing package if the existing package is of same version as that of the framework package version. Ofcourse, registering existing package in this case is more efficient but adding the package again will cover more issues with the existing package.